.github/workflows/build-wheel-linux-x86_64.yaml

```diff
@@ -136,7 +136,7 @@ jobs:
 
     - name: Install Dependencies (Python)
       run: |
-        python${{ matrix.python_version }} -m pip install wheel numpy pybind11 PyYAML
+        python${{ matrix.python_version }} -m pip install wheel numpy pybind11 PyYAML scipy
```

Comment:
Just double checking because these are build-time dependencies. Is `scipy` a built-time dependency if we are going the dynamic loading approach?